### ISSUE TITLE
Logic Analyzer exports empty CSV

### ISSUE THREAD

**Issue by pcercuei** [analogdevicesinc/scopy#666]
After capturing data, I set "Export All" to ON state, click Export, and save the result file as "test.csv" in my home directory.

But the only content of the file is:
```
;date mar. août 6 17:59:58 2019
;version Scopy - 48fb6a9
;comment 2945024 samples acquired at 2.94118e+6 Hz 
```

Scopy 1.0.6 on Debian (flatpak).

**Comment by AlexandraTrifan** [comment]
Should be fixed in this release, using the latest flatpak: https://github.com/analogdevicesinc/scopy/releases/tag/v1.2.0-rc.1

**Comment by tagoylo** [comment]
This bug is fixed in v1.2. Closing this item.
